src/builtin_types.md

```diff
@@ -47,7 +47,7 @@ enum Void {
 }
 ```
 
-The type of `void`, also known as "unit" or "nothing".
+The type of `void`, Grain's "unit" type.
```

Comment:
We may need to explain this a little better for people not familiar with the concept of unit.

Reply:
It might help linking to wikipedia, https://en.wikipedia.org/wiki/Unit_type for the unit part. But I think we could better clarify that void holds no information to improve the users understanding of void.